src/client/table_client.rs

```diff
@@ -1520,6 +1492,12 @@ fn build_runtime(name: &str, threads_num: usize) -> runtime::Runtime {
 
 fn build_obkv_runtimes(config: &ClientConfig) -> ObClientRuntimes {
     ObClientRuntimes {
+        batch_op_runtime: Arc::new(build_runtime(
+            "ob-batch-executor",
+            config.batch_op_thread_num,
+        )),
+        query_runtime: Arc::new(build_runtime("ob-query-executor", config.query_thread_num)),
+        conn_init_runtime: Arc::new(build_runtime("ob-conn-initer", config.conn_init_thread_num)),
```

Comment:
Maybe what we need here is just a `bg_runtime` to do some work including connection initialization, table location refreshing?

And replace `default_runtime` with the `bg_runtime`.

Reply:
Fixed. We only keep three necessary runtimes for different tasks. See [ObClientRuntimes](https://github.com/oceanbase/obkv-table-client-rs/pull/52/files#diff-5d22ee0b1e6b8fa1bedfc0c5f66207feb1b680ce55f03796505bdba35ad96fc9)